docs/docs/marker.mdx

```diff
@@ -0,0 +1,280 @@
+# Markers docs
+
+# Markers
```

Comment:
Do we need to mention it's an experimental feature anywhere?

Reply:
Now added.